### DISCUSSION TITLE
PKsim Liver compartment question

### DISCUSSION THREAD

**Discussion by jyna6** [Open-Systems-Pharmacology/Forum#1675] [Questions & Problems]
I am now doing some simulations in PKsim platform and wonder what each liver compartment means.
For example, there is a "Liver- Liver Periportal" option and a "Liver Periportal - Plasma" option.
What would be the difference between those two??

**Comment by StephanSchaller** [comment]
Please check the docu here:
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-creating-individuals#editing-and-scaling-individual-properties

--> scroll up a bit or search the page for "zonation".

**Reply by StephanSchaller** [reply]
Now in updated Doc:
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-creating-individuals#anatomy-and-physiology --> scroll down to "Liver Zonation"

**Reply by jmgallo** [reply]
It is still confusing and it might present an issue with body weight. For example, if you enter 30gm mouse as desired individual and examine organ volumes you'll find both liver and liver periportal weight = 1.86e-3 L (no zonation).  However, when you sum all the organ weight including the mucosal the total body weight without the liver periportal weight = 27.7gm. I don't believe the liver periportal wight should be included in the total body weight since it is already depicted as a single liver compartment. If you add in the liver periportal weight then the total BW = 29.5gm. Please clarify. thanks

**Reply by StephanSchaller** [reply]
I don't think the liver periportal is added. Only the liver, which is already a sum of periportal + central.

**Reply by jmgallo** [reply]
I agree it shouldn't be added (unless you select zonation); however, without the addition of the liver periportal the sum of all the organ weights including mucosa does not = 30gm. Am I missing something? thanks

**Reply by StephanSchaller** [reply]
are the Blood Organs included (Venous Arterial blood)?

**Reply by jmgallo** [reply]
Yes.

**Reply by Christoph27** [reply]
For a 30 g mouse without liver zonation the weight of perportal liver is 1.86e-3 g which is also the weight of total liver (no zonation) and the weight of the pericentral liver is 0 accordingly. Below you can see what is summed up to get the 30 g body weight (you can get every equation after exporting a PK-Sim simulation with right mouse click and "Export Simulation Structure to File" or alternatively by exporting to MoBi ).  

Parameter: sim|Organism|Weight of blood organs
	IsFixed: False
	Value: 0.00128932463947456 [kg]
	Formula: P_1 + P_2 + P_3
		P_1: sim|Organism|VenousBlood|Weight (tissue)(0.000749347484650897 [kg])
		P_2: sim|Organism|ArterialBlood|Weight (tissue)(0.000325803626671743 [kg])
		P_3: sim|Organism|PortalVein|Weight (tissue)(0.00021417352815192 [kg])

Parameter: sim|Organism|Weight of tissue organs
	IsFixed: False
	Value: 0.0287106753605254 [kg]
	Formula: P_1 + P_2 + P_3 + P_4 + P_5 + P_6 + P_7 + P_8 + P_9 + P_10 + P_11 + P_12 + P_13 + P_14 + P_15 + P_16
		P_1: sim|Organism|Bone|Weight (tissue)(0.00225596116320023 [kg])
		P_2: sim|Organism|Brain|Weight (tissue)(0.000242729998572177 [kg])
		P_3: sim|Organism|Fat|Weight (tissue)(0.0014278235210128 [kg])
		P_4: sim|Organism|Gonads|Weight (tissue)(0.000356955880253201 [kg])
		P_5: sim|Organism|Heart|Weight (tissue)(0.000135643234496216 [kg])
		P_6: sim|Organism|Kidney|Weight (tissue)(0.000485459997144353 [kg])
		P_7: sim|Organism|Stomach|Weight (tissue)(0.000157060587311408 [kg])
		P_8: sim|Organism|SmallIntestine|Weight (tissue)(0.002007519870544 [kg])
		P_9: sim|Organism|LargeIntestine|Weight (tissue)(0.000895245347675027 [kg])
		P_10: sim|Organism|Liver|Periportal|Weight (tissue)(0.00185617057731664 [kg])
		P_11: sim|Organism|Liver|Pericentral|Weight (tissue)(0 [kg])
		P_12: sim|Organism|Lung|Weight (tissue)(0.00014278235210128 [kg])
		P_13: sim|Organism|Muscle|Weight (tissue)(0.014278235210128 [kg])
		P_14: sim|Organism|Pancreas|Weight (tissue)(0.000185617057731664 [kg])
		P_15: sim|Organism|Skin|Weight (tissue)(0.00414068821093713 [kg])
		P_16: sim|Organism|Spleen|Weight (tissue)(0.00014278235210128 [kg])

Parameter: sim|Organism|Weight
	IsFixed: False
	Value: 0.03 [kg]
	Formula: Weight_blood + Weight_tissue
		Weight_blood: sim|Organism|Weight of blood organs(0.00128932463947456 [kg])
		Weight_tissue: sim|Organism|Weight of tissue organs(0.0287106753605254 [kg])

**Comment by jmgallo** [comment]
Hi,
I appreciate your answer, but there is a difference in the total BW as you outlined in your note and what I extracted from PK-Sim for a 30 gm mouse (from excel file), right? You can see where I subtracted the liver periportal weight from the totals since we shouldn't count it twice. I also included the mucosa weights. Please let me know if you find any errors, but as I see it a 30 gm BW mouse is not 100% accounted for in the weight distribution.
Thanks, jim




Mouse 30 gm - from PK-Sim
BW = 30 gm
Anatomy
Organ Volumes (L)
Liters
venous blood
7.49E-04
arterial blood
3.26E-04
bone
2.26E-03
brain
2.43E-04
fat
1.43E-03
gonads
3.57E-04
heart
1.36E-04
kidney
4.85E-04
large intestine
8.95E-04
liver
1.86E-03
liver periportal
1.86E-03
lung
1.43E-04
muscle
1.00E-02
pancreas
1.86E-04
portal vein
2.10E-04
skin
4.14E-03
small intestine
2.01E-03
spleen
1.43E-04
stomach
1.57E-04
Organ total Volume
2.76E-02
organ total volume - liver periportal
2.57E-02
Mucosa
duodenum
2.38E-04
upper jejunum
5.41E-04
lower jejunum
5.00E-04
upper ileum
6.31E-05
lower ileum
6.23E-05
cecum
2.00E-04
colon ascendens
8.18E-05
colon transversum
8.18E-05
colon desendens
5.46E-05
colon sigmoid
5.46E-05
rectum
5.46E-05
mucosa sum
1.93E-03
total body weight
2.95E-02
total BW - liver periportal
2.77E-02


Jim Gallo
Empire Innovation Professor
Department of Pharmaceutical Sciences
University at Buffalo
Email: ***@***.******@***.***>

**Reply by Yuri05** [reply]
Here you can see which organs contribute to the body weight. You should NOT add mucosa because it's already included into the total weight of small/large intestine organs.
![grafik](https://github.com/user-attachments/assets/fe7b1583-4e4f-4262-8135-b1e03b29bdd2)

So the body weight is the sum of: 
<img width="400" src="https://github.com/user-attachments/assets/a720f1a4-45a8-47eb-8640-cb811adde8e7">

Which for the 30g mouse sums up to ... 30g 😃 
<img width="200" src="https://github.com/user-attachments/assets/76925fdd-5c3e-450e-b810-214b33ab5b46">

**Comment by jmgallo** [comment]
I appreciate your help. It seems the differences are primarily due to round-off or significant digits. In PK-Sim muscle weight is given as 0.01L (my initial value) but if you expand it gives 1.43L (your value) that accounts for the differences (portal vein was off a little too).
Thanks


Jim Gallo
Empire Innovation Professor
Department of Pharmaceutical Sciences
University at Buffalo
Email: ***@***.******@***.***>
